packages/hagrid/hagrid/cli.py

```diff
@@ -67,6 +71,8 @@ def get_azure_image(short_name: str) -> str:
 
 
 @click.group(cls=RichGroup)
+# experimenting with click.option
```

Comment:
Remove unused commented lines to keep our PRs to only include changes that are being used.

Reply:
Okay, thanks Madhava!